earth_enterprise/src/third_party/skia/SConscript

```diff
@@ -100,6 +100,8 @@ sgl_env.UpdateCppflagsForSkia()
 
 cxx_flags = sgl_env['CXXFLAGS']
 cxx_flags.remove('-Werror')
+cxx_flags.append('-fPIC')
```

Comment:
Not sure what it is for, can you update the issue or description with some explanation?